In [3]:
#Import Library
import pandas as pd

In [4]:
filepath1 = '/content/drive/MyDrive/Coding Dojo/Project 2/Advertising Budget and Sales.csv'
filepath2= '/content/drive/MyDrive/Coding Dojo/Project 2/Company Bankrucptcy Prediction .csv'

In [5]:
df1=pd.read_csv(filepath1)
df2=pd.read_csv(filepath2)

In [7]:
#Dataset 1
df1.head()
df1.shape


(200, 1)

#Explanation About Dataset 1 - Advertising and Sales
1) Source of data
     https://www.kaggle.com/datasets/yasserh/advertising-sales-dataset
2) Brief description of data
      The Data is about advertising budget and total sales.
3) What is the target?
     The target is total sales
4) Is this a classification or regression problem?
     Regression Problems
5) How many features?
     4 Fatures
6) How many rows of data.
     200 rows and 4 columns
7) What, if any, challenges do your foresee in cleaning, exploring, or modeling with this dataset?

In [9]:
df2=pd.read_csv(filepath2)
df2
df2.shape


(6819, 96)

#Explanation About Dataset 2 - Company Bankruptcy
Source of data https://www.kaggle.com/datasets/fedesoriano/company-bankruptcy-prediction

1.   Source of data https://www.kaggle.com/datasets/
2.   Brief description of data:  The Data is financial ratio of company that can be used to determine profitability, leverage, liquidity of a company. 
3.  What is the target? The target is to predict bankcruptcy of a company based on the financial ratio.
4. Is this a classification or regression problem?
      Regression Problem
5. How many features? 96 Fatures 
6) How many rows of data?. 6819 rows and 4 columns 
7) What, if any, challenges do your foresee in cleaning, exploring, or modeling with this dataset?